internal/sysfs/sysfs_test.go

```diff
@@ -68,6 +68,13 @@ func testOpen_Read(t *testing.T, tmpDir string, testFS FS) {
 		require.Equal(t, syscall.ENOENT, err)
 	})
 
+	t.Run("doesn't exist", func(t *testing.T) {
+		_, err := testFS.OpenFile(path.Join(dir, "/really-not-exist"), os.O_RDONLY, 0)
+
+		// We currently follow os.Open not syscall.Open, so the error is wrapped.
```

Comment:
I think the assertion is verifying this isn't wrapped though ;) (which is congruent with the other functions so correct IMHO). Maybe change or delete the comment?

Reply:
oh this is the garbage and not representative. WIll just delet.